_sass/components/sidebar.scss

```diff
@@ -70,6 +70,22 @@
     }
   }
 
+      &:hover {
```

Comment:
It looks like these are supposed to be inside another selector but they're not. This is preventing the branch from building for me.